modules/cnf-configuring-nrop-on-schedlable-control-planes.adoc

```diff
@@ -0,0 +1,224 @@
+
+When you configure control plane nodes to be schedulable, you can optionally configure the NUMA Resources Operator to manage them. To do this, define separate `nodeGroups` for the control plane and worker nodes in the NUMAResourcesOperator custom resource (CR)
+
+This configuration allows the NUMA Resources Operator to schedule pods on both control plane and worker nodes, taking into consideration resource availability and NUMA topology. The NUMA Resources Operator treats Control plane nodes like any other schedulable node on the cluster, and they can be included in the `nodeGroups` for Topology-Aware Scheduling (TAS).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NUMA Resources Operator.
+
+.Procedure
+
+. To enable Topology Aware Scheduling (TAS) on control plane nodes, configure the nodes to be schedulable first. This allows the NUMA Resources Operator to deploy and manage pods on them. Without this action, the operator cannot deploy the pods required to gather NUMA topology information from these nodes. Follow these steps to make the control plane nodes schedulable:
+
+.. Edit the `schedulers.config.openshift.io` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit schedulers.config.openshift.io cluster
+----
+
+.. In the editor, set the `mastersSchedulable` field to `true`, then save and exit the editor.
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Scheduler
+metadata:
+  creationTimestamp: "2019-09-10T03:04:05Z"
+  generation: 1
+  name: cluster
+  resourceVersion: "433"
+  selfLink: /apis/config.openshift.io/v1/schedulers/cluster
+  uid: a636d30a-d377-11e9-88d4-0a60097bee62
+spec:
+  mastersSchedulable: true
+status: {}
+#...
+----
+
+. Configure the NUMA Resources Operator to manage the schedulable control plane nodes. The `nodeGroups` configuration in the NUMAResourcesOperator CR tells the Operator which nodes to manage. There must be only one `numaresourcesoperator` resource on the cluster.
+
+.. For a Compact Cluster, the compact cluster's master nodes are also the schedulable nodes, so specify only the master pool. Create the following `nodeGroups` configuration in the `NUMAResourcesOperator` CR:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+    - poolName: master
+----
++
+[NOTE]
+====
+Using the worker pool in addition to the master pool in a compact cluster can create issues with duplicate pods. 
+====
+
+.. For an MNO custer, since both the control plane and worker nodes are schedulable, specify both the master and worker pools. Create the following `nodeGroups` configuration in the `NUMAResourcesOperator` CR:
```

Comment:
If there is no infra reasons for having all the nodes enabled with NROP, I don't imagine any customer wouldn't want to run TAS on all of the nodes, however the configuration of the CR depends on what the user wants. instead of assuming that the user want to include all MCPs, perhaps highlight that multiple node groups can be specified as part of the CR, leaving the option of the configuration to the user.